src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -5285,6 +5393,9 @@ export class StudyViewPageStore {
     readonly sampleMutationCountAndFractionGenomeAlteredData = remoteData({
         await: () => [this.clinicalAttributes, this.selectedSamples],
         invoke: () => {
+            if (this.selectedSamples.result.length === 0) {
+                return new Promise(() => []) as Promise<Array<ClinicalData>>;
```

Comment:
`Promise.resolve([])`, I'm pretty sure what you have now would never resolve because the promise signature is 
```
Promise((resolve, reject)=>{
   // do something where you call the `resolve` or `reject` callbacks with the object to be returned
})
```

Reply:
Good call.

Reply:
Is there a reason why we didn't notice this in app?  I mean the whole never resolving business.  Maybe because selectedSamples.result was never zero length?